go/libraries/doltcore/sqle/dfunctions/dolt_merge.go

```diff
@@ -47,101 +47,110 @@ type DoltMergeFunc struct {
 const DoltMergeWarningCode int = 1105 // Since this our own custom warning we'll use 1105, the code for an unknown error
 
 const (
-	hasConflictsOrViolations int = 0
-	noConflictsOrViolations  int = 1
+	noConflictsOrViolations  int = 0
+	hasConflictsOrViolations int = 1
+)
+
+const (
+	threeWayMerge = 0
+	ff            = 1
 )
 
 func (d DoltMergeFunc) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 	args, err := getDoltArgs(ctx, row, d.Children())
 	if err != nil {
 		return noConflictsOrViolations, err
 	}
-	return DoDoltMerge(ctx, args)
+	hasConflicts, _, err := DoDoltMerge(ctx, args)
+	return hasConflicts, err
```

Comment:
so we only get the `fast_forward` info through the store procedure variant?

Reply:
correct